### ISSUE TITLE
Breaks when no stack trace is available

### ISSUE THREAD

**Issue by emhagman** [jbielick/faktory_worker_node#29]
https://github.com/jbielick/faktory_worker_node/blob/a86d1df9c62bf9acc5f7ff75a4e6dea244e31fc3/lib/client.js#L242

I haven't found the culprit error that has no stack trace, but I am getting the following error:

```
TypeError: Cannot read property 'split' of undefined
    at Client.fail (/app/node_modules/faktory-worker/lib/client.js:242:28)
    at Worker.handle (/app/node_modules/faktory-worker/lib/worker.js:247:25)
```

**Comment by jbielick** [comment]
That's unfortunate 😖 . Thanks for the issue. I've addressed this in d2a093 and published @ 3.0.2

**Comment by emhagman** [comment]
Thanks!
